README.md

```diff
@@ -212,6 +215,22 @@ Response:
 }
 ```
 
+### Filtering Subtitles
+
+If you'd like to filter subtitles, one way to do so is by using regexing. asbplayer will match any sequence following regex pattern and remove the matches.
+
+Under the MISC section in asbplayer settings, locate the "Subtitle regex filter" textbox. Enter an appropriate regex to filter desired content.
+You can replace filtered content similarly by entering a string into the "Subtitle regex filter text replacement" textbox. Leaving this blank will simply remove the content.
+
+Potential use cases for filtering include:
+ - Remove names enclosed by parenthesis to indicate speakers  (i.e. "**(山田)**　元気ですか？")
+ - Remove indications enclosed by brackets that sound or music that is playing (i.e. "**\[PLAYFUL MUSIC]**")
+
+Helpful materials on regexing:

```

Comment:
Same comment as about "regexing." I think it's clearer/more helpful to say "writing regular expressions" or similar.